cms/djangoapps/contentstore/helpers.py

```diff
@@ -299,13 +302,21 @@ def import_staged_content_from_user_clipboard(parent_key: UsageKey, request) ->
             tags=user_clipboard.content.tags,
         )
 
+        usage_key = new_xblock.scope_ids.usage_id
```

Comment:
```suggestion
        usage_key = new_xblock.usage_key
```
If you want, there is now a simpler way to get this :)

Reply:
Updated https://github.com/openedx/edx-platform/pull/36058/commits/a254fe4a5672e796baf46c3caa40d569a0203133